package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@curi-bio/mantarray-frontend-components",
-  "version": "1.2.4",
+  "version": "1.2.6",
```

Comment:
You don't have to bump this until we decide to publish it

Reply:
It was 1.2.4 before so I changed it to that